forms/AvatarInput/index.js

```diff
@@ -20,15 +20,12 @@ module.exports = React.createClass({
   },
 
   getDefaultProps: function() {
-    global.ENV = global.ENV || {};
-
     return {
       options: {
         imageDim: [600, 600],
         cropRatio: 1 / 1,
         mimetype: ['image/*'],
-        services: ['CONVERT', 'COMPUTER'],
-        debug: global.ENV.TEST_MODE
```

Comment:
So, I think that var was used to allow some HUI integration test to reach in and add an image without driving the UI. I’m assume you are removing it because that is not the case anymore?